ui/app/AppLayouts/Chat/views/ChatMessagesView.qml

```diff
@@ -155,7 +155,7 @@ Item {
 
         onContentYChanged: {
             scrollDownButton.visible = contentHeight - (d.scrollY + height) > 400
-            if(d.scrollY < 500) messageStore.loadMoreMessages()
+            if(d.scrollY < 1000) messageStore.loadMoreMessages()
```

Comment:
Have you tested the app in the minimum size? Won't it result in infinite messages loading?

Reply:
I just did. It won't result in infinite messages loading as it depends on the model size. Isn't perfect as it is still flooding the messageStore with loadMoreMessages and I was looking into a protection for the mssageStore. But I've seen your PR and it will fix it very nicely (#8854)

Reply:
> It won't result in infinite messages loading as it depends on the model size

Sorry, I should be more precise, by infinite I meant: won't it load too many (maybe all) messages from database without user interaction?

Anyway, after second thought, it probably won't, because the scroll gets smaller with the higher number of messages loaded, meaning `scrollY` can only grow without user interaction.

Reply:
![image](https://user-images.githubusercontent.com/33099791/212681166-cd778e94-6aea-412f-9cc9-348f9802f196.png)

A picture is worth a thousand words

Reply:
Indeed, there is a flood of requests that can be triggered either by scroll, chat height change or content height change. The loadMoreMessages function is called multiple times per second because it depends on these variables.
There are two triggers:
1. UI interaction (scroll - whenever the user scrolls to the top 1000 px; app size change - if the visible view is in the top 1000 px of the listview).  Here I've noticed the flood happening. 
2. Model changes (whenever the number of items changes and the visible view is in the top 1000 px)

I haven't noticed any symptoms in the app because the nim message service has a protection against this flood and it won't process any new loadMoreMessages until there is an answer from the current request.
It's something not that obvious and I'm glad you have spotted it. Your PR will fix this, right?

Here is a short demo, but notice there is no backend request going out:

https://user-images.githubusercontent.com/47811206/212693167-296e1e98-2f67-4c45-9428-5632df56cedc.mov

Sorry for the long explanation, but it's for me to make sure I understood the point because having infinite and recursive requests would be really bad.

Reply:
False alarm, `scrollY` is not what I thought, it takes into account content height. We are ok!